packages/flutter_tools/pubspec.yaml

```diff
@@ -11,17 +11,17 @@ dependencies:
   # To update these, use "flutter update-packages --force-upgrade".
   archive: 2.0.10
   args: 1.5.2
-  dwds: 0.6.2
+  dwds: 0.7.1
```

Comment:
Did you need to pull in the change from https://github.com/flutter/flutter/pull/41805/files#diff-ce38e968fc555ba18c509b8fac294f0aR266 for this

Reply:
Done.